lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js

```diff
@@ -0,0 +1,65 @@
+import moment from 'moment'
+import { formatDuration } from '@opentripplanner/core-utils/lib/time'
+
+/**
+ * Calculate commonly-used pieces of data used to render the trip status
+ * component. The monitoredTrip param can be undefined.
+ */
+export default function baseRenderer (monitoredTrip) {
+  const data = {
+    // the threshold for schedule deviation in seconds for whether an arrival or
+    // departure time is to be considered on-time
+    // TODO: get this from config or something
+    ON_TIME_THRESHOLD_SECONDS: 120,
+    // create some default display values in case another renderer doesn't
+    // calculate these values
+    body: 'Unkown trip state',
+    headingText: 'Unkown trip state',
```

Comment:
Typos 'Unknown'... The spellchecker didn't catch these?

Reply:
That is unfortunate that the spellchecker didn't catch these. I'll add it to my todo-list to explore other spellcheckers. For now, I fixed this in 3a36d3983359b49803f6cd8df4a6d2eb49c83251